composer.json

```diff
@@ -19,15 +19,6 @@
         "wpackagist-plugin/*",
         "wpackagist-theme/*"
       ]
-    },
-    {
```

Comment:
Should we leave this as-is, to make it easier to install these addons when someone needs them?

Reply:
So just remove the require rows? Can do.

Reply:
Yes. Then it's trivial to just run
```
composer require whatever/from/monni
```

instead of first adding the new repo.